templates/paragraphs/paragraph--oe-timeline.html.twig

```diff
@@ -4,8 +4,12 @@
  * Theme override to display the 'timeline' type paragraph.
  */
 #}
+{% if content.field_oe_text_long|field_value|render is not empty %}
+  {% set introduction = content.field_oe_text_long|field_value|render %}
+{% endif %}
```

Comment:
Looking around I think we can follow the example to put this into a variable and check that if it's not empty and then assign ti to the render array. My concern here is we call render twice here.